Test-NetStack.psm1

```diff
@@ -716,15 +716,13 @@ Function Test-NetStack {
                     }
                 }
             }
-
```

Comment:
So minor nit: Why did we remove these line breaks here? Reads better with a break after a curly.